includes/class-wc-stripe-webhook-handler.php

```
@@ -524,11 +524,30 @@ public function process_webhook_dispute_closed( $notification ) {
 	 * @param object $notification
 	 */
 	public function process_webhook_capture( $notification ) {
-		$order        = WC_Stripe_Helper::get_order_by_charge_id( $notification->data->object->id );
+		$charge       = $notification->data->object;
+		$order        = WC_Stripe_Helper::get_order_by_charge_id( $charge->id );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'process_webhook_capture'. (undefined)

(StaticAccess)